zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -4224,21 +4265,38 @@ components:
           $ref: "#/components/schemas/SortOrderEnum"
       required:
         - field
-    UserTaskSearchQueryRequest:
-      allOf:
-        - $ref: "#/components/schemas/SearchQueryRequest"
-      description: User task search query request.
+    UserTaskSearchQueryRequestBase:
+      description: Base properties for UserTaskSearchQueryRequest
       type: object
       properties:
         sort:
           description: Sort field criteria.
           type: array
           items:
             $ref: "#/components/schemas/UserTaskSearchQuerySortRequest"
+    UserTaskSearchQueryRequest:
+      description: User task search query request.
+      deprecated: true
+      type: object
+      allOf:
+        - $ref: "#/components/schemas/SearchQueryRequest"
+        - $ref: "#/components/schemas/UserTaskSearchQueryRequestBase"
+      properties:
         filter:
           description: The user task search filters.
           allOf:
             - $ref: "#/components/schemas/UserTaskFilterRequest"
+    UserTaskSearchQueryRequestBody:
+      description: User task search query request. Key attributes as string values.
+      type: object
+      allOf:
+        - $ref: "#/components/schemas/SearchQueryRequest"
```

Comment:
❓ Could this also move into the `UserTaskSearchQueryRequestBase`?

Reply:
Yes, it actually could.